conf/engaging.config

```
@@ -1,7 +1,7 @@
 // Nextflow config for running on the MIT Engaging HPC cluster
 params {
     config_profile_description = 'MIT Engaging HPC cluster profile.'
-    config_profile_contact     = 'Phil Palmer (@PhilPalmer)'
+    config_profile_contact     = 'Charlie Whittaker (@Charlie14557807)'
```

```suggestion
    config_profile_contact     = 'Charlie Whittaker (@Charlie14557807)'
```
github.com/Charlie14557807 throws a 404, is this maybe a twitter username?

---

yes... sorry i was not sure what to put there but understand now, ill fix it